packages/now-cli/src/util/dev/server.ts

```diff
@@ -1130,6 +1140,19 @@ export default class DevServer {
     }
   }
 
+  getExtensionlessFile = (path: string) => {
+    const ext = extname(path);
+    if (
+      this.apiDir &&
+      path.startsWith(this.apiDir + '/') &&
```

Comment:
Will this be correct on Windows?

Reply:
I guess I could add windows to the `now dev` CI 🤔

Reply:
I confirmed this works with Windows. I also created PR #3727 to add Windows to CI.